resources/packs/azure/azure.lr

```diff
@@ -19,9 +19,11 @@ azuread.user {
   accountEnabled bool
   mailNickname string
   mail string
-  objectType string
+  // deprecated, no longer used
+  objectType() string
```

Comment:
I did not drop the deprecated fields as that's breaking change but rather made them a function. Semantically it's all the same, but that allows us to throw an error when computing the value.